src/fa.c

```diff
@@ -4513,6 +4513,88 @@ void fa_dot(FILE *out, struct fa *fa) {
     fprintf(out, "}\n");
 }
 
+void fa_json(FILE *out, struct fa *fa) {
+    int *list_hashes;
```

Comment:
Initialize this to `NULL`. We won't hit the error label right now before we try to allocate, but it makes it safer in case we ever change the code so that it could encounter an error before allocation.